src/App.js

```diff
@@ -1,13 +1,68 @@
 import React from 'react';
 
 import './App.scss';
+import { GoodsList } from './components';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  handleGetGoods = () => (
+    getAll()
+      .then((goods) => {
+        this.setState({ goods });
+      })
```

Comment:
Potentially it can be taken out to some method, like `updateGoods` and used in all three handlers.